src/Main/Main.js

```diff
@@ -81,15 +88,38 @@ export class Main extends Component {
               </ul> : <span>Posts are empty</span>
             }
             <Button />
-            <GeoBtn />
             <UserTasks />
           </div>
           <div className="box_2">
             {this.state.show && <Timer />}
             <button onClick={() => this.setState({ show: false })}>Destroy</button>
           </div>
+          <div className="box_3">
+            <EditableComponent onLoose={text => console.log(text)} />
+            <TodoList />
+            <Geo />
+          </div>
+          <div className="box_4">
+            <Tabs tabs={tabs} />
+          </div>
         </div>
       </main>
     );
   }
 }
+UserTasks.propTypes = {
+  tasks: PropTypes.number,
+  done: PropTypes.number,
+  inProgress: PropTypes.number,
+  waiting: PropTypes.number
+};
+Button.propTypes = {
+  addClass: PropTypes.bool,
+  hidden: PropTypes.bool
+};
+Main.propTypes = {
+  users: PropTypes.array,
+  posts: PropTypes.array,
+  show: PropTypes.bool
+};
+
```

Comment:
For some reason you're using state as a props. Where did you get this?
state is defined in component `this.state = {...}`

and props are defined when you create component. i.e. <User properties... />
`<Comp prop1="one" prop2="two" />` --> this.props ==> { prop1: "one", prop2: "two" }

propTypes and defaultProps are only for props, state shouldn't be used inside type checking